.github/workflows/create-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Create new release
+
+on:
+  # run when a release is published
+  # triggers for both prerelease and release
+  release:
+    types:
+    - published
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # required to access tags
+          submodules: 'true'
+
+      - name: Build & run devcontainer image
+        # this always builds a new image from scratch rather than from the build-devcontainer-image workflow output
+        # so that we pick up the latest versions of everything
+        id: devcontainer
+        run: |
+          docker build --tag devcontainer:latest .devcontainer
+          container_id=$(docker create -w /workspace -v $GITHUB_WORKSPACE:/workspace -v /var/run/docker.sock:/var/run/docker.sock devcontainer:latest)
+          docker start "$container_id"
+          echo "::set-output name=id::$container_id"
+        
+      - name: Login to registry
```

Comment:
Does the `docker/login-action` not work for us here for some reason?

Reply:
I ask just because that's what we use in the other files

Reply:
The login has to happen inside the (dev)container, not on the host.